### ISSUE TITLE
Accessibility issues

### ISSUE THREAD

**Issue by mindctrl** [restrictcontentpro/restrict-content-pro#1394]
A customer emailed these suggestions to us to fix some accessibility issues on the registration page.

> First item: 

> The list of form error messages is represented in <p> paragraph elements. For a11y, these should be semantically represented in a <ul> unordered list. Or, if the elements cannot be changed, use ARIA roles:

> `<div class="rcp_message" role="list">`
> `<p class="rcp_error" role="listitem">`

> Second item:

> Since the error messages are dynamically added to the page without refresh, the "rcp_message error" div needs to receive focus so that screen readers go straight to the error message and read it instead of waiting on the submit button with no indicators anything on page has changed.

> When errors are present, in addition to the error message above the submit, an error message should appear in the label of the input that contains an error.

> When submit button is pressed without filling out any information, no individual fields are in error but should be.

**Comment by nosegraze** [comment]
> When submit button is pressed without filling out any information, no individual fields are in error but should be.

Can anyone else reproduce this? I get normal error messages if I just leave everything blank and click submit.

**Comment by mindctrl** [comment]
I think he's saying the fields themselves don't get error indicators. The errors are all detached from the thing that has an error.
